src/pocketmine/network/mcpe/protocol/BossEventPacket.php

```diff
@@ -39,9 +39,9 @@ class BossEventPacket extends DataPacket{
 	public const TYPE_HIDE = 2;
 	/* C2S: Unregisters a player from a boss fight. */
 	public const TYPE_UNREGISTER_PLAYER = 3;
-	/* S2C: Appears not to be implemented. Currently bar percentage only appears to change in response to the target entity's health. */
+	/* S2C: Currently bar percentage only appears to change in response to the target entity's health. */
```

Comment:
This seems to need further work.

Reply:
oh, my bad, just a quick delete didn't work